controllers/apps/transformer_component_workload.go

```diff
@@ -644,19 +644,20 @@ func (r *componentWorkloadOps) leaveMember4ScaleIn() error {
 	}
 
 	var leaveErrors []error
+	var leaveDone []string
 	for _, pod := range podsToMemberLeave {
 		if podsToMemberjoin.Has(pod.Name) {
-			leaveErrors = append(leaveErrors, fmt.Errorf("pod %s is in memberjoin process", pod.Name))
+			r.reqCtx.Log.Info(fmt.Sprintf("pod %s is in memberjoin process", pod.Name))
 			continue
 		}
-		if err := r.leaveMemberForPod(pod, pods); err != nil {
+		if err = r.leaveMemberForPod(pod, pods); err != nil {
 			leaveErrors = append(leaveErrors, err)
 		}
 	}
 	if len(leaveErrors) > 0 {
-		return newRequeueError(time.Second, fmt.Sprintf("%v", leaveErrors))
+		return nil, newRequeueError(time.Second, fmt.Sprintf("%v", leaveErrors))
 	}
-	return nil
+	return leaveDone, nil
```

Comment:
`leaveDone` seems to be always empty, with no elements inserted into it. Also, does this issue exist in KB 1.0 as well?